featuretools/tests/conftest.py

```diff
@@ -470,3 +470,46 @@ def koalas_entities():
 @pytest.fixture
 def relationships():
     return [("cards", "id", "transactions", "card_id")]
+
+
+@pytest.fixture(params=['pd_transform_es', 'dask_transform_es', 'koalas_transform_es'])
+def transform_es(request):
+    return request.getfixturevalue(request.param)
+
+
+@pytest.fixture
+def pd_transform_es():
+    # Create dataframe
+    df = pd.DataFrame({'a': [14, 12, 10], 'b': [False, False, True],
+                       'b1': [True, True, False], 'b12': [4, 5, 6],
+                       'P': [10, 15, 12]})
+    es = ft.EntitySet(id='test')
+    # Add dataframe to entityset
+    es.entity_from_dataframe(entity_id='first', dataframe=df,
```

Comment:
I think this is ok for now since I don't think we are using these fixtures in the integration yet, but these will need to be updated to `es.add_dataframe`...

Reply:
I'll create an issue for tracking

Reply:
Filed #1495